src/java/org/apache/cassandra/index/sai/plan/Expression.java

```diff
@@ -169,6 +172,7 @@ public Expression add(Operator op, ByteBuffer value)
             case EQ:
             case CONTAINS:
             case CONTAINS_KEY:
+            case IN:
```

Comment:
Hmm, looking at the usages of `isEquality`, it's probably not, but technically, it is equality, and it reads a bit weird.

Reply:
Yeah, because `isEquality()` is used when we actually hit the index, it doesn't matter right now. However, it would probably make sense to add it, because if we ever do support it at the index level, we'll need it. @sunil9977 Does that make sense? I don't think it actually breaks anything in the meantime...

Reply:
Got it. Marking IN as equality for future index-level support makes sense. No impact for now, so we can track this as needed.